packages/react-core/src/components/Breadcrumb/BreadcrumbItem.tsx

```diff
@@ -38,7 +49,8 @@ export const BreadcrumbItem: React.FunctionComponent<BreadcrumbItemProps> = ({
           <AngleRightIcon />
         </span>
       )}
-      {to && (
+      {to && render && render({ href: to, children, target, isActive, showDivider, ...props })}
```

Comment:
I think it would be more useful to use the render prop to return things like the PF classname and some aria attributes, like this:
```
<BreadcrumbItem isActive render={({ className, ariaCurrent }) => {
  <Link to="your-link" className={className} aria-current={ariaCurrent}>Router item</Link>
}} />
```

and in the code (but you'd want to hold classname etc in variables so it is not duplicated across the code)
```
render && render({ className: css(styles.breadcrumbLink, isActive && styles.modifiers.current), ariaCurrent: isActive ? 'page' : undefined,...props })}
```